### DISCUSSION TITLE
Lightweight web interface ?

### DISCUSSION THREAD

**Discussion by alexandre-romain** [ggml-org/llama.cpp#1391] [Q&A]
Got LLAMA + the 65b models running (slowly, but usable) on a Debian 11 Hyper-V VM running on a dual Xeon 5218R HP Proliant DL385 server (SSD disks, 128Go RAM)

I would like to be able to use it through a 'ChatGPT-like' interface but didn't find any 'out of the box' solution that doesn't require tons of dependencies ...

Anything you could link me to ?

Thanks a lot !

**Comment by ido-pluto** [comment]
You can check out [CatAI](https://github.com/ido-pluto/catai). It is very simple to install.

It also has a chatGPT like ui.
checkout the [additional commands](https://github.com/ido-pluto/catai/blob/main/docs/commands.md) for that.

**Comment by nuance1979** [comment]
You can also try [llama-server](https://github.com/nuance1979/llama-server).

**Comment by SlyEcho** [comment]
Or [koboldcpp](https://github.com/LostRuins/koboldcpp)
